js/aspnet.js

```diff
@@ -143,17 +143,29 @@
     function createComponent(name, options, id, validatorOptions) {
         var selector = '#' + String(id).replace(/[^\w-]/g, '\\$&');
         pendingCreateComponentRoutines.push(function() {
-            var $component = $(selector)[name](options);
-            if($.isPlainObject(validatorOptions)) {
-                $component.dxValidator(validatorOptions);
+            var $element = $(selector);
+            if($element.length) {
+                var $component = $(selector)[name](options);
+                if($.isPlainObject(validatorOptions)) {
+                    $component.dxValidator(validatorOptions);
+                }
+                return true;
             }
+            return false;
         });
     }
 
     templateRendered.add(function() {
         var snapshot = pendingCreateComponentRoutines.slice();
+        var leftover = [ ];
+
         pendingCreateComponentRoutines = [ ];
-        snapshot.forEach(function(func) { func(); });
+        snapshot.forEach(function(func) {
+            if(!func()) {
+                leftover.push(func);
```

Comment:
Can we push items to `pendingCreateComponentRoutines` directly?
```suggestion
                pendingCreateComponentRoutines.push(func);
```